plugins/org.eclipse.n4js.flowgraphs/src/org/eclipse/n4js/flowgraphs/analysis/EdgeGuide.java

```diff
@@ -8,7 +8,7 @@
  * Contributors:
  *   Marcus Mews - Initial API and implementation
  */
-package org.eclipse.n4js.flowgraphs.analyses;
+package org.eclipse.n4js.flowgraphs.analysis;
```

Comment:
why rename "org.eclipse.n4js.flowgraphs.analyses" to "org.eclipse.n4js.flowgraphs.analysis"?
For packages "org.eclipse.n4js.flowgraphs.dataflow.guards" and "org.eclipse.n4js.flowgraphs.dataflow.symbols" you also use the plural form.

Reply:
Actually, I am not sure. I think I renamed this because `analysis` is easer to distinguish from `analysers`, compared to distinguishing `analyses` from `analysers`.
I also see it analog to the package `validation` (would map to `analysis`) which provides the framework, and `validators` (would map to `analysers`) which uses the framework by implementing specific analyses.